internal/testing/releasehelper.go

```diff
@@ -132,24 +153,32 @@ func runOnGomod(path string) {
 				rel, _ = filepath.Split(rel)
 			}
 
-			if *doReplace {
-				cmdCheck(fmt.Sprintf("go mod edit -replace=%s=%s %s", r.Path, rel, gomodPath))
-			} else if *doDropReplace {
-				cmdCheck(fmt.Sprintf("go mod edit -dropreplace=%s %s", r.Path, gomodPath))
-			}
+			reqHandler(gomodPath, r.Path, rel)
 		}
 	}
 }
 
 func main() {
-	flag.Usage = func() {
```

Comment:
Consider using `github.com/google/subcommands`. It does a better job at `help` than you, and will set up the commands as struct that can define their own flags, etc.

https://github.com/google/go-cloud/blob/master/samples/gocdk-secrets/main.go has a simple example.

Reply:
I considered it, and I'd rather not. Right now these simple helper tools are single-file. If they grow dependencies they'll need to be split into separate modules (or some unified `tools` module). 

Moreover, it's not clear to me what the advantage of `subcommands` is for this specific tool. It seems pretty trivial to me :man_shrugging:  Commands can have their own flags with `flag.NewFlagSet` if needed.

Besides, we use cobra for the gocdk which makes sense, but how many different subcommand packages we really need in a single project? I'd vote to only use cobra for complicated stuff like the CLI and the standard library for everything else.